.github/workflows/kube-test.validate-kind-cluster.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ -z "${GITHUB_ACTIONS+x}" ] ; then
+  exit 1
+fi
+
+if [ "$#" -ne 1 ] ; then
+  echo "$0 supports exactly 1 argument"
+fi
+
+kube_version="${1}"
+
+test_script="
+set -o errexit
+set -o pipefail
+set -o nounset
+
+kind create cluster \
+  --config /etc/kind/${kube_version}/standard-github-actions-cluster.yaml \
+  --kubeconfig /github/workspace/kubeconfig
+
+kubectl get nodes --kubeconfig /github/workspace/kubeconfig --context kind-kind --output wide
+kubectl apply --kubeconfig /github/workspace/kubeconfig --context kind-kind --filename https://raw.github.com/cilium/cilium/v1.8.0/install/kubernetes/quick-install.yaml
```

Comment:
Is there a better way for managing which branch/tag we should take `quick-install.yaml` from? Maybe an env variable/script argument?

Reply:
This script is only to ensure that `kube-tester` image doesn't break. It doesn't matter which version of Cilium is install, we could actually skip installing Cilium here, but the built-in bridge CNI is already disabled, so some network provider is needed.